_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -100,10 +100,18 @@ This `input` element is not [focusable][] because of the `disabled` attribute.
 This `a` element is not [focusable][] because it moves focus to the `input` element whenever it receives focus.
```

Comment:
```suggestion
This `a` element is not [focusable][] because it moves focus to the `button` element whenever it receives focus.
```

Reply:
Focus moves to the input (top of the dialog) and not the button.